core/templates/domain/utilities/stopwatch.model.ts

```diff
@@ -46,17 +43,8 @@ export class Stopwatch {
     }
     return (this._getCurrentTime() - this.startTime) / 1000;
   }
-}
 
-@Injectable({
-  providedIn: 'root'
-})
-export class StopwatchObjectFactory {
-  create(): Stopwatch {
+  static create(): Stopwatch {
```

Comment:
Ditto here and elsewhere, can we have this as a constructor?

Reply:
Using the constructor instead of create is causing some tests to fail for this file.

Reply:
You may have to change the test as per this new structure. Can you please share the issue you are facing, we can help if fix it (if required)? 

/cc @srijanreddy98